webserver/server/app/models/pagination/slotBoundsPagination.sql

```diff
@@ -0,0 +1,25 @@
+/* @name slotBoundsPagination */
+WITH MIN_HASH AS
+	(SELECT COALESCE("Transaction".ID,
+
+										-1) AS MIN_TX_ID,
+			SLOT AS MIN_SLOT
+		FROM "Transaction"
+		JOIN "Block" ON "Block".ID = "Transaction".BLOCK_ID
+		WHERE SLOT <= :low!
+		ORDER BY "Block".ID DESC, "Transaction".ID DESC
+		LIMIT 1),
+	MAX_HASH AS
+	(SELECT SLOT AS MAX_SLOT,
+			COALESCE(MAX("Transaction".ID),
+
+				-2) AS MAX_TX_ID
+		FROM "Transaction"
+		JOIN "Block" ON "Transaction".BLOCK_ID = "Block".ID
+		WHERE SLOT <= :high!
+		GROUP BY "Block".ID
+		ORDER BY "Block".ID DESC
+		LIMIT 1)
+SELECT *
+FROM MIN_HASH
+LEFT JOIN MAX_HASH ON 1 = 1;
```

Comment:
The formatting on this file is really strange making it hard to read. Here is an updated formatting

```suggestion
 /* @name slotBoundsPagination */
WITH
min_hash AS
(
         SELECT   COALESCE("Transaction".id, -1) AS min_tx_id,
                  slot                           AS min_slot
         FROM     "Transaction"
         JOIN     "Block"
         ON       "Block".id = "Transaction".block_id
         WHERE    slot <= :low!
         ORDER BY "Block".id DESC,
                  "Transaction".id DESC
         LIMIT 1
),
max_hash AS
(
         SELECT   slot                                AS max_slot,
                  COALESCE(Max("Transaction".id), -2) AS max_tx_id
         FROM     "Transaction"
         JOIN     "Block"
         ON       "Transaction".block_id = "Block".id
         WHERE    slot <= :high!
         GROUP BY "Block".id
         ORDER BY "Block".id DESC
         LIMIT 1
)
SELECT    *
FROM      min_hash
LEFT JOIN max_hash
ON        1 = 1; 
```

Reply:
was this manual formatting, btw? I originally was using the one from pgadmin, but I agree that it was weird. I just was hesitant to change it since it was automatic.